packages/framework/src/Resources/config/services.yml

```diff
@@ -102,8 +102,6 @@ services:
   Shopsys\FrameworkBundle\Component\Breadcrumb\BreadcrumbResolver:
     factory: ['@Shopsys\FrameworkBundle\Model\Breadcrumb\FrontBreadcrumbResolverFactory', create]
 
-  Shopsys\FrameworkBundle\Model\Breadcrumb\FrontBreadcrumbResolverFactory: ~
```

Comment:
How did you find out which services to remove?

Reply:
Removed services did not have configuration and each of them suited autodiscovery pattern.